lib/bolt/cli.rb

```diff
@@ -559,6 +563,40 @@ def initialize_project
       ok ? 0 : 1
     end
 
+    def migrate_project
+      if inventory.version == 2
+        ok = true
+      else
+        inventoryfile = config.inventoryfile || config.boltdir.inventory_file
+        inv = YAML.load_file(inventoryfile)
+
+        inv = Bolt::Util.walk_pairs(inv) do |k, v|
+          if k == 'nodes'
+            k = 'targets'
+            v = v.map do |target|
+              target['uri'] = target.delete('name') if target.is_a? Hash
+              target
+            end
+          end
+
+          { k => v }
+        end
+
+        inv['version'] = 2
```

Comment:
Looks like this will print `version:2` at the very bottom of the file. If you did something like
```
 ok = File.write(inventoryfile, {'version' => 2}.merge(inv).to_yaml)
```
I think it would print at the top.